osu.Game/Screens/Ranking/ScorePanelList.cs

```diff
@@ -117,19 +118,24 @@ public ScorePanel AddScore(ScoreInfo score)
                 d.Origin = Anchor.Centre;
             }));
 
-            if (SelectedScore.Value == score)
-                selectedScoreChanged(new ValueChangedEvent<ScoreInfo>(SelectedScore.Value, SelectedScore.Value));
-            else
+            if (IsLoaded)
```

Comment:
I'd probably invert this check, although that's sort of subjective.

Reply:
I dunno, not a huge fan of the double return..

![image](https://user-images.githubusercontent.com/191335/97657639-c0213c00-1aad-11eb-9c01-d8f858966552.png)